Code/Framework/AzCore/Tests/Memory.cpp

```diff
@@ -2557,88 +2034,63 @@ namespace UnitTest
             printf("\n\t\t\t=======================\n");
             printf("\t\t\tSchemas Benchmark Test!\n");
             printf("\t\t\t=======================\n");
-            DebugSysAllocSchema da;
-            {
-                HphaSchema::Descriptor hphaDesc;
-                hphaDesc.m_fixedMemoryBlockByteSize = AZ_TRAIT_OS_HPHA_MEMORYBLOCKBYTESIZE;
-                hphaDesc.m_fixedMemoryBlock = AZ_OS_MALLOC(hphaDesc.m_fixedMemoryBlockByteSize, hphaDesc.m_fixedMemoryBlockAlignment);
-                HeapSchema::Descriptor dlMallocDesc;
-                dlMallocDesc.m_numMemoryBlocks = 1;
-                dlMallocDesc.m_memoryBlocksByteSize[0] = hphaDesc.m_fixedMemoryBlockByteSize;
-                dlMallocDesc.m_memoryBlocks[0] = AZ_OS_MALLOC(dlMallocDesc.m_memoryBlocksByteSize[0], hphaDesc.m_fixedMemoryBlockAlignment);
-                PoolSchema::Descriptor poolDesc;
-                poolDesc.m_pageAllocator = &da;
-                poolDesc.m_numStaticPages = 100;
-                printf("\t\t\tPrealocated memory %.2f MB!\n", (float)hphaDesc.m_fixedMemoryBlockByteSize / (1024.f*1024.f));
+            {
+                // TODO Switch to using instance of HphaAllocator with a sub allocator of a fixed size
```

Comment:
TODO: Add a GHI issue ticket for making this change, so that the work can be added to the backlog